### PR TITLE
Renaming of pyMANGA 

### PR DESCRIPTION
**Pull request by jbathmann**
With this commit the old name (mangrove-froundwater feedback model) is updated to "PYthon Models for AGeNt-based resource GAthering"

Adressess #200

### CODE REVIEW

**Comment by jbathmann** [comment]
Deploy preview worked after restart:

https://deploy-preview-201--pymanga.netlify.app/

File: web/content/de/_index.html [hunk 1]
```diff
@@ -16,15 +16,16 @@
...
+Describing vegetation population dynamics based on first principles
 
 {{% /blocks/lead %}}
 
 {{< blocks/section color="primary" >}}
 
 <font size='5'> Für weitere Details zu dem Projekt lesen Sie die letzte Publikation: </font>
 
+Bathmann, J., Peters, R., Reef, R., Walther, M., Lovelock, C. E., 2021. Modelling mangrove forest structure and species composition over tidal inundation gradients: The feedback between plant water use and porewater salinity in an arid mangrove ecosystem. Agricultural and Forest Meteorology 308-309(1650):108547
 Bathmann, J., Peters, R., Naumov, D., Fischer, T., Berger, U., Walther, M., 2020. The MANgrove–GroundwAter feedback model (MANGA) – Describing belowground competition based on first principles. Ecological Modelling 11.
```
Comment (mcwimm) [inline web/content/de/_index.html:30]: Hier fehlt der Zeilenumbruch zwischen den Publikationen.

File: web/content/de/_index.html [hunk 2]
```diff
@@ -16,15 +16,16 @@
...
+PYthon Models for AGeNt-based resource GAthering (pyMANGA) 
 
-Describing belowground competition based on first principles
+Describing vegetation population dynamics based on first principles
 
 {{% /blocks/lead %}}
 
 {{< blocks/section color="primary" >}}
 
 <font size='5'> Für weitere Details zu dem Projekt lesen Sie die letzte Publikation: </font>
```
Comment (mcwimm) [inline web/content/de/_index.html:27]:
```suggestion
<font size='5'> Für weitere Details zu dem Projekt lesen Sie die aktuellen Publikationen: </font>
```

File: web/content/en/_index.html [hunk 3]
```diff
@@ -27,5 +27,6 @@
 
 <font size='5'> If you want to know more details about the project see also the recent publication: </font>
```
Comment (mcwimm) [inline web/content/en/_index.html:28]:
```suggestion
<font size='5'> If you want to know more details about the project see also the recent publications: </font>
```

File: web/content/en/_index.html [hunk 4]
```diff
@@ -27,5 +27,6 @@
...
 
+Bathmann, J., Peters, R., Reef, R., Walther, M., Lovelock, C. E., 2021. Modelling mangrove forest structure and species composition over tidal inundation gradients: The feedback between plant water use and porewater salinity in an arid mangrove ecosystem. Agricultural and Forest Meteorology 308-309(1650):108547
 Bathmann, J., Peters, R., Naumov, D., Fischer, T., Berger, U., Walther, M., 2020. The MANgrove–GroundwAter feedback model (MANGA) – Describing belowground competition based on first principles. Ecological Modelling 11.
```
Comment (mcwimm) [inline web/content/en/_index.html:31]: Zeilenumbruch (wie dt. Seite)

**Review by mcwimm — COMMENTED** [review]
👍

File: web/content/de/_index.html (hunk 1 above)
Comment (jbathmann) [inline web/content/de/_index.html:30]:
```suggestion

Bathmann, J., Peters, R., Naumov, D., Fischer, T., Berger, U., Walther, M., 2020. The MANgrove–GroundwAter feedback model (MANGA) – Describing belowground competition based on first principles. Ecological Modelling 11.
```

Comment (jbathmann) [reply web/content/de/_index.html:30]: Hinzugefügt

File: web/content/en/_index.html (hunk 4 above)
Comment (jbathmann) [inline web/content/en/_index.html:31]:
```suggestion

Bathmann, J., Peters, R., Naumov, D., Fischer, T., Berger, U., Walther, M., 2020. The MANgrove–GroundwAter feedback model (MANGA) – Describing belowground competition based on first principles. Ecological Modelling 11.
```

**Comment by jbathmann** [comment]
Suggestions added.

@mcwimm Please approve this PR, if it is ready.

**Review by mcwimm — APPROVED** [review]
👍

**Pull request merged**